### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by MMAThijssen**
Added code to build Docker image and deploy on the GCP.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM python:3.9-slim-buster
+
+EXPOSE 8080
+
+COPY requirements_dev.txt ./requirements_dev.txt
```
Comment (andrewsutjahjo) [inline Dockerfile:5]: This should be `requirements.txt` as we're not going to be dev soon :)

File: requirements_dev.txt [hunk 2]
```diff
@@ -7,5 +7,6 @@ tox==3.14.0
 coverage==4.5.4
 Sphinx==1.8.5
 twine==1.14.0
-Click==7.1.2
+Click==7.0
+Flask==1.1.2
```
Comment (andrewsutjahjo) [inline requirements_dev.txt:11]: This needs to go into `requirements.txt` as well

**Review by andrewsutjahjo — CHANGES_REQUESTED** [review]
Small change needed in the requirements. 

Also, the docker image is `python==3.9` right now the CI/CD is set up with `python==3.7.x` or `python==3.8.x` in mind. It does double-check if things pass with `python==3.9.x` but it isn't a hard requirement. 
I'm sure there won't be any clashes, but could you double-check if the rest is developing on python 3.9 just in case?

**Comment by MMAThijssen** [comment]
Made the changes as requested. Use python 3.8 as basis now.

**Review by andrewsutjahjo — APPROVED** [review]
Nice one!

**Pull request merged**